go.mod

```diff
@@ -33,7 +33,7 @@ require (
 	github.com/elastic/gosigar v0.14.2
 	github.com/etclabscore/go-openrpc-reflect v0.0.36
 	github.com/fatih/color v1.15.0
-	github.com/filecoin-project/filecoin-ffi v0.30.4-0.20220519234331-bfd1f5f9fe38
```

Comment:
I was scratching my head about this being so old until I remembered about the `replace`, so this isn't technically needed but certainly nice to get it off 2022! can't wait for ffi go binary modules tho.

Reply:
@rvagg I don't think I will be working on ffi go modules anytime soon man. It is a non trivial amount of work and I would like to focus on the ETH/Fevm side of things.

Reply:
If someone else wants to hack on it -> happy to share my findings/direct them to the docs we've produced for it.